recipes/install.rb

```diff
@@ -92,7 +92,7 @@
     path '/etc/systemd/system/mesos-slave.service'
     source 'systemd.erb'
   when 'sysvinit_debian'
-    mode 0755
+    mode 0o755
```

Comment:
I haven't seen mode represented in this way before.  Is this valid?  What's the point of this change?